candle-nn/src/optim.rs

```diff
@@ -190,4 +190,12 @@ impl AdamW {
         };
         Self::new(vars, params)
     }
+
+    pub fn params(&self) -> ParamsAdamW {
```

Comment:
What about returning a reference instead so that it's up to the user to decide whether they want to clone it or not?

Reply:
Thank you. Updated.